ballerina-shell/modules/shell-cli/src/test/resources/testcases/operations.immutable.json

```diff
@@ -19,21 +19,6 @@
     "description": "Define immutable Student &",
     "code": "Student & readonly student = { 'class: 12, details: immutableDetails, marks: { math: 80, physics: 85, chemistry: 75 } };"
   },
-  {
```

Comment:
https://github.com/ballerina-platform/ballerina-lang/issues/33789 tracks these. We may not need tests specifically for `isReadOnly()` though, because we are planning on removing it for GA.